src/main/java/pro/sky/telegrambot/servise/NotificationServise.java

```diff
@@ -0,0 +1,66 @@
+package pro.sky.telegrambot.servise;
+
+import com.pengrad.telegrambot.TelegramBot;
+import com.pengrad.telegrambot.request.SendMessage;
+import com.pengrad.telegrambot.response.SendResponse;
+import pro.sky.telegrambot.model.NotificationTask;
+import org.springframework.stereotype.Service;
+import pro.sky.telegrambot.repository.NotificationRepository;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+
+@Service
+public class NotificationServise {
+    private final NotificationRepository notificationRepository;
+    private final TelegramBot telegramBot;
+
+    public NotificationServise(NotificationRepository notificationRepository, TelegramBot telegramBot) {
+        this.notificationRepository = notificationRepository;
+        this.telegramBot = telegramBot;
+    }
+
+    public NotificationTask add(NotificationTask notificationTask) {
+        return notificationRepository.save(notificationTask);
+    }
+
+    public Collection<NotificationTask> getAll() {
+
+        return notificationRepository.findAll();
+    }
+
+
+    public void SendNotification() {
```

Comment:
```suggestion
    public void sendNotification() {
```
Названия методов исключительно с маленькой буквы